src/store/model-store.js

```diff
@@ -69,6 +70,15 @@ store.registerHandler('DELETE_ITEM', data => {
 });
 
 store.registerHandler('NEW_MODEL', data => {
+    assert.isString(data.name, 'The model name must be a string');
```

Comment:
It's an assertion library, not just one for testing. This prevents the client from creating an invalid model and ruining the state on the server.